src/sass/base/dsf-fonts.scss

```diff
@@ -1,3 +1,116 @@
 // -----------------------------------------------------------------------------
 // This file contains all @font-face declarations, if any.
-// -----------------------------------------------------------------------------
\ No newline at end of file
+// -----------------------------------------------------------------------------
+
+// Roboto Thin
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-display: swap;
+  src: url("../../../build/fonts/Roboto-Thin.ttf") format("truetype");
```

Comment:
I think for this we should use the google CDN demo'd here:
https://fonts.google.com/specimen/Roboto

This will allow for improved caching and more lightweight delivery for our package (and we don't have to manage the files! :) )

Reply:
fixed